### PR TITLE
First draft of qotd tutorial

### PR DESCRIPTION
**Pull request by prakhar1989**

### CODE REVIEW

File: qotd/blog.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
+# Motivational QOTD with Courier and GPT2
+
+## Background
```
Comment (shreythecray) [inline qotd/blog.md:3]: I'd recommend adding either screenshots/gifs or embedding your demo video at the top.

File: qotd/blog.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+Motivational quotes were rage back in the day with MMS & email forwards were a thing. I remember my parents forwarding me one every morning so that I can start my day with one. Fast forward to today, if you are lucky, you are part of some forward group on your messaging app of choice (Whatsapp, Telegram etc.).
```
Comment (shreythecray) [inline qotd/blog.md:5]: Love this!

File: qotd/blog.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
+# Motivational QOTD with Courier and GPT2
+
+## Background
+
+Motivational quotes were rage back in the day with MMS & email forwards were a thing. I remember my parents forwarding me one every morning so that I can start my day with one. Fast forward to today, if you are lucky, you are part of some forward group on your messaging app of choice (Whatsapp, Telegram etc.). 
+
+Inspired by the same idea, today we are going to build a service that sends our friends and family an AI generated motivational quote of the day. Rather than hardcoding a list of motivational quotes, we are going to use a machine learning model to generate a quote on demand. That way we know that we'll never run out of quotes to share!
+
+![what we will build](samples/email.png)
```
Comment (shreythecray) [inline qotd/blog.md]: Image isn't showing up

File: qotd/blog.md [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+
+## Instructions
+
+### Part 1: Using AI to generate motivational quotes
+
+#### OpenGPT2 and Language Models
+OpenAI GPT-2 model was proposed in Language Models are Unsupervised Multitask Learners by Alec Radford, Jeffrey Wu, Rewon Child, David Luan, Dario Amodei and Ilya Sutskever. It’s a causal transformer pretrained using language modeling on a very large corpus of ~40 GB of text data.
```
Comment (shreythecray) [inline qotd/blog.md:16]: Cool background!

File: qotd/blog.md [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+In this project, we'll use the [HuggingFace](https://huggingface.co/) library to load and serve the ML model that will generate the quotes for us. HuggingFace makes it very easy to use transformer models (of which GPT2 is a type) in our projects without any knowledge of ML or AI. As mentioned earlier, GPT2 is a general purpose language model which means that is has is good at predicting generic text given an input sequence. In our case, we need a model more suited for generating quotes. To do that, we have two options
+
+1) We can fine-tune the GPT2 model by using our own text for which we'll need a good dataset of quotes.
+2) Or we can find an *already* existing model which has been fine-tuned with some quotes.
+
+Luckily, in our case there exists a fine-tuned model that has been trained on the 500k quotes dataset - https://huggingface.co/nandinib1999/quote-generator
+
+With HuggingFace, using this model is as easy as 
+
+```python
```
Comment (shreythecray) [inline qotd/blog.md:33]: Can we break down this code a bit more?

File: qotd/blog.md [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+
+And that's it!
+
+## Conclusions
+
+This tutorial demonstrated how easy it is to get started with machine learning & Courier.
+
+If you want to go ahead and improve this project, here are some interesting ideas to try
+
+- Better background image: Use a term from the generated quote to search for an image?
```
Comment (shreythecray) [inline qotd/blog.md:247]: This is awesome! Overall, great blogpost. I'm sure our community would love to build this project themselves.

File: qotd/blog.md [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+
+## Background
+
+Motivational quotes were rage back in the day with MMS & email forwards were a thing. I remember my parents forwarding me one every morning so that I can start my day with one. Fast forward to today, if you are lucky, you are part of some forward group on your messaging app of choice (Whatsapp, Telegram etc.). 
+
+Inspired by the same idea, today we are going to build a service that sends our friends and family an AI generated motivational quote of the day. Rather than hardcoding a list of motivational quotes, we are going to use a machine learning model to generate a quote on demand. That way we know that we'll never run out of quotes to share!
+
+![what we will build](samples/email.png)
+
+## Instructions
```
Comment (shreythecray) [inline qotd/blog.md:11]: Are there any pre-reqs required to build this? Level of coding experience needed?

**Review by shreythecray — CHANGES_REQUESTED** [review]

File: qotd/blog.md (hunk 7 above)
Comment (prakhar1989) [reply qotd/blog.md:11]: Nope, no per-requisites needed to follow along this tutorial.

File: qotd/blog.md (hunk 6 above)
Comment (prakhar1989) [reply qotd/blog.md:247]: Thank you! Glad to hear that!

File: qotd/blog.md (hunk 5 above)
Comment (prakhar1989) [reply qotd/blog.md:33]: Done - clarified this a bit more.

File: qotd/blog.md (hunk 4 above)
Comment (prakhar1989) [reply qotd/blog.md:16]: 💯

File: qotd/blog.md (hunk 2 above)
Comment (prakhar1989) [reply qotd/blog.md:5]: Thank you!

File: qotd/blog.md (hunk 3 above)
Comment (prakhar1989) [reply qotd/blog.md]: Fixed

File: qotd/blog.md (hunk 1 above)
Comment (prakhar1989) [reply qotd/blog.md:3]: I've added the screenshot of the email. Let me know if more screenshots are needed.

**Pull request merged**